sklearn/random_projection.py

```diff
@@ -547,15 +553,16 @@ class SparseRandomProjection(BaseRandomProjection):
         Use density = 1 / 3.0 if you want to reproduce the results from
         Achlioptas, 2001.
 
-    eps : strictly positive float, optional, (default=0.1)
+    eps : float, default=0.1
+        Strictly positive.
         Parameter to control the quality of the embedding according to
         the Johnson-Lindenstrauss lemma when n_components is set to
         'auto'.
-
+    
```

Comment:
```suggestion

```
To fix `sklearn/random_projection.py:561:1: W293 blank line contains whitespace`